warehouse/oidc/models/_core.py

```diff
@@ -174,20 +174,10 @@ class OIDCPublisherMixin:
     # but there are a few problems: those claim sets don't map to their
     # "equivalent" column (only to an instantiated property), and may not
     # even have an "equivalent" column.
```

Comment:
I think this comment is now invalid and probably should be written to express the new design of "implement the checks in a subclass' `lookup_by_claims()`